_config.yml

```diff
@@ -1,8 +1,8 @@
 title: P-society
 email: psocietyiiitbh@gmail.com
-description: We are a bunch of undergrads who like to dabble into all things in computer science from
-             web dev to machine learning and from distributed systems to blockchain,
-             helping others along the way in kick-starting their journey of becoming a developer.
+description: We are a bunch of undergraduate students who like to dabble into all things computer science from
+             web development, to machine learning, to distributed systems, to blockchain,
```

Comment:
so many `to's` can you phrase it any other way?

Reply:
I'll look into it.